src/DatabaseLibrary/connection_manager.py

```diff
@@ -74,63 +136,68 @@ def connect_to_database(self, dbapiModuleName=None, dbName=None, dbUsername=None
+                dbconnection = db_api_2.connect('DRIVER={SQL Server};SERVER=%s,%s;DATABASE=%s;UID=%s;PWD=%s' % (dbHost, dbPort, dbName, dbUsername, dbPassword))
+            elif dbapiModuleName in ["excel"]:
+                logger.info(
+                    'Connecting using : %s.connect(DRIVER={Microsoft Excel Driver (*.xls, *.xlsx, *.xlsm, *.xlsb)};DBQ=%s;ReadOnly=1;Extended Properties="Excel 8.0;HDR=YES";)' % (
+                    dbapiModuleName, dbName))
+                dbconnection = db_api_2.connect(
+                    'DRIVER={Microsoft Excel Driver (*.xls, *.xlsx, *.xlsm, *.xlsb)};DBQ=%s;ReadOnly=1;Extended Properties="Excel 8.0;HDR=YES";)' % (
+                        dbName), autocommit=True)
+            elif dbapiModuleName in ["excelrw"]:
+                logger.info(
+                    'Connecting using : %s.connect(DRIVER={Microsoft Excel Driver (*.xls, *.xlsx, *.xlsm, *.xlsb)};DBQ=%s;ReadOnly=0;Extended Properties="Excel 8.0;HDR=YES";)' % (
+                    dbapiModuleName, dbName))
+                dbconnection = db_api_2.connect(
+                    'DRIVER={Microsoft Excel Driver (*.xls, *.xlsx, *.xlsm, *.xlsb)};DBQ=%s;ReadOnly=0;Extended Properties="Excel 8.0;HDR=YES";)' % (
+                        dbName), autocommit=True)
+            elif dbapiModuleName in ["ibm_db", "ibm_db_dbi"]:
+                dbPort = dbPort or 50000
+                logger.info('Connecting using : %s.connect(DATABASE=%s;HOSTNAME=%s;PORT=%s;PROTOCOL=TCPIP;UID=%s;PWD=%s;) ' % (dbapiModuleName, dbName, dbHost, dbPort, dbUsername, dbPassword))
+                dbconnection = db_api_2.connect('DATABASE=%s;HOSTNAME=%s;PORT=%s;PROTOCOL=TCPIP;UID=%s;PWD=%s;' % (dbName, dbHost, dbPort, dbUsername, dbPassword), '', '')
+            elif dbapiModuleName in ["cx_Oracle"]:
+                dbPort = dbPort or 1521
+                oracle_dsn =  db_api_2.makedsn(host=dbHost, port=dbPort, service_name=dbName)
+                logger.info('Connecting using: %s.connect(user=%s, password=%s, dsn=%s) ' % (dbapiModuleName, dbUsername, dbPassword, oracle_dsn))
+                dbconnection = db_api_2.connect(user=dbUsername, password=dbPassword, dsn=oracle_dsn)
+            else:
+                logger.info('Connecting using : %s.connect(database=%s, user=%s, password=%s, host=%s, port=%s) ' % (dbapiModuleName, dbName, dbUsername, dbPassword, dbHost, dbPort))
+                dbconnection = db_api_2.connect(database=dbName, user=dbUsername, password=dbPassword, host=dbHost, port=dbPort)
+
+
+            self._cache.register(dbconnection, alias=alias)
+
+        except Exception as Err:
+            err_msg = ('DbConnection : %s : %s' % (alias,Err))           
+            raise AssertionError(err_msg)
+            
+
+    def connect_to_database_using_custom_params(self, alias, dbapiModuleName=None, db_connect_string=''):
```

Comment:
I don't see the point of duplicating the method, logging can be added anyway in the called method

Reply:
with only one connection there would be no need. With multiple connections I thought it was more correct to add to the error message also the name of the connection that produced it.